### PR TITLE
Use crypto-secure PRNG for all parts of password

### LINKED ISSUE #8: Use crypto-secure random number generation
According to this discussion thread, the nice cryptographically secure random number functions KeePass provides are only used to seed .NET's built-in random number generator: http://sourceforge.net/p/kpwordsequence/discussion/general/thread/f472c3eb/

This makes me nervous. The .NET PRNG is not cryptographically secure; it could potentially introduce predictable patterns in the generated passwords.

Maybe I'm being paranoid, but I'm going to hold off on using these passwords for anything sensitive until good crypto is used throughout.

### PR DESCRIPTION
**Pull request by fritzophrenic**
Fixes #8

### CODE REVIEW

File: ItemGenerator/WordGenerator.cs [hunk 1]
```diff
@@ -23,12 +22,10 @@ public string Generate(WordSequenceItem item, int seed)
                 if (item.Words == null || !item.Words.Override)
                     wordList.AddRange(_globalConfiguration.DefaultWords);
 
-                targetWord = wordList[randomSeed.Next(wordList.Count)];
+                targetWord = wordList[cryptoRandom(0, wordList.Count-1)];
```
Comment (fireout) [inline ItemGenerator/WordGenerator.cs]: This confuses me, does it compile? (I can not try it myself yet)
Either missing the `.GetRandomInRange` or delegate definition, altough if a delegate is defined, might as well use it throughout...

File: Sequencer.cs [hunk 2]
```diff
@@ -123,32 +123,35 @@ public void Save(PasswordSequenceConfiguration configuration)
...
         }
 
-        public string GenerateSequence(PasswordSequenceConfiguration globalConfiguration, Random randomSeed)
+        public string GenerateSequence(PasswordSequenceConfiguration globalConfiguration, CryptoRandomRange cryptoRandom)
         {
             string targetSequence = string.Empty;
             foreach (SequenceItem sequenceItem in globalConfiguration.Sequence)
-                if (sequenceItem.Probability != PercentEnum.Never && randomSeed.Next(101) <= (int)sequenceItem.Probability)
-                    targetSequence += GenerateSequenceItem(sequenceItem, globalConfiguration, randomSeed);
+                if (sequenceItem.Probability != PercentEnum.Never && (int)cryptoRandom.GetRandomInRange(0, 100) <= (int)sequenceItem.Probability)
```
Comment (fireout) [inline Sequencer.cs]: Should probably split the line in two (I know it wasn't done before :/ ) as you did mostly everywhere else; ie: better readability

File: ItemGenerator/WordGenerator.cs (hunk 1 above)
Comment (fritzophrenic) [inline ItemGenerator/WordGenerator.cs]: I'm confused, too. I did compile and test before pushing...but I agree that
should not work. Maybe my commit --amend screwed something up. Honestly I'm
constantly worried I'm doing git wrong. I'll look into that today.

Comment (fritzophrenic) [inline ItemGenerator/WordGenerator.cs]: I fixed this issue. It still compiled and ran with a full rebuild, even before I fixed it...I don't know why.

File: Sequencer.cs (hunk 2 above)
Comment (fritzophrenic) [inline Sequencer.cs]: Done, and in a few other places as well.

**Pull request merged**